docs/dev-guide/iframe.md

```diff
@@ -390,68 +515,96 @@ api.executeCommand('startRecording', {
 });
 ```
 
-* **stopRecording** - Stops an ongoing recording, `stream` or `file`.
+* **stopRecording** - Stops an ongoing **`stream`** or **`file`** recording.
+
+No other params are required.
+
 ```javascript
 api.executeCommand('stopRecording', 
     mode: string //recording mode to stop, `stream` or `file`
 );
 ```
 
-You can also execute multiple commands using the `executeCommands` method:
+## **Multiple command execution**
+
+You can also execute multiple commands using the **`executeCommands`** method:
+
 ```javascript
 api.executeCommands(commands);
 ```
-The `commands` parameter is an object with the names of the commands as keys and the arguments for the commands as values:
+
+The **`commands`** parameter is an object with the names of the commands as keys and the arguments for the commands as values:
+
 ```javascript
 api.executeCommands({
     displayName: [ 'nickname' ],
     toggleAudio: []
 });
 ```
 
-You can add event listeners to the embedded Jitsi Meet using the `addEventListener` method.
-**NOTE: This method still exists but it is deprecated. JitsiMeetExternalAPI class extends [EventEmitter]. Use [EventEmitter] methods (`addListener` or `on`).**
+## **Event listeners**
+You can add event listeners to the embedded Jitsi Meet using the **`addEventListener`** method.
+
+**NOTE:** The **`addEventListener`** still exists but it is deprecated.  The **`JitsiMeetExternalAPI`** class extends [EventEmitter]. Use [EventEmitter] methods (e.g., **`addListener`** or **`on`**).
```

Comment:
This has been deprecated for a while, let's drop it.

Reply:
Remove the complete Event listener section?

Reply:
removed all event listener content

Reply:
Not all, we need that, just the deprecation note.

Reply:
Removed note